resources/tentative/perf.webkit.org/public/v2/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
This file can probably be removed since the test directly links to v3

Reply:
Removed.

Reply:
Removed offsetX/offsetY.